src/lib/sidenav/drawer.ts

```diff
@@ -359,6 +383,32 @@ export class MdDrawerContainer implements AfterContentInit {
         this._watchDrawerPosition(drawer);
       });
     });
+
+    const resize = typeof window !== 'undefined' ?
+      auditTime.call(fromEvent(window, 'resize'), 150) :
+      observableOf(null);
+
+    this._updateDrawer = startWith.call(resize, null).subscribe(() => {
+      if (this._element.nativeElement.offsetWidth < this.breakpointWidth) {
```

Comment:
I haven't checked, but this needs to be consistent with any other responsive behavior in this lib or in flex-layout (i.e. `<` vs `<=`)